### PR TITLE
Update TFM_PlayerListener.java

### LINKED ISSUE #98: EventHandling (If someone says "can i have op", auto-opall)
title says it

### PR DESCRIPTION
**Pull request by DUFCLiam**
Closes #98

### CODE REVIEW

**Comment by LegendIsAwesomes** [comment]
Why not just check who is online, if they arent op, op them

**Comment by DUFCLiam** [comment]
Ok, Sure :)

**Comment by DUFCLiam** [comment]
@LegendIsAwesomes

**Comment by LegendIsAwesomes** [comment]
I dont want them to say I want op, I want it to auto check without them asking for op, so they are auto opped.

**Comment by DUFCLiam** [comment]
Better @LegendIsAwesomes

File: src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java [hunk 1]
```diff
@@ -820,6 +820,12 @@ public void onPlayerJoin(PlayerJoinEvent event)
                 TFM_AdminList.updateLastLogin(player);
             }
         }
+        // Handles OP on join
+            if (!player.isOP())
```
Comment (LegendIsAwesomes) [inline src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java]: A untested PR, doesn't compile

Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java]: I had to quickly do it online editor, I will fix it now :)

**Comment by LegendIsAwesomes** [comment]
Never do that.

**Comment by DUFCLiam** [comment]
Ok, I will fix it right now got the correct code I think.

**Comment by DUFCLiam** [comment]
@LegendIsAwesomes Should fix it.

File: src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java [hunk 2]
```diff
@@ -820,6 +820,13 @@ public void onPlayerJoin(PlayerJoinEvent event)
                 TFM_AdminList.updateLastLogin(player);
             }
         }
+        // Handles OP on join
+            Player player = event.getPlayer;
+            if (player.isOp() == false)
+            {
+                player.setOP(true);
```
Comment (LegendIsAwesomes) [inline src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java]: Again, the capitalization if the OP, Not allowing it to compile.

**Comment by LegendIsAwesomes** [comment]
Dont say you forgot, you added it yourself.

**Comment by DUFCLiam** [comment]
Better? xD @LegendIsAwesomes

**Pull request merged**